Example/URBNDataSource/ViewControllers/Code CollectionView/URBNSwiftAccordionTableViewController.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  URBNSwiftAccordionTableViewController.swift
+//  URBNDataSource
+//
+//  Created by Dustin Bergman on 12/15/15.
+//  Copyright © 2015 Joe. All rights reserved.
+//
+
+import UIKit
+
+typealias CellTappedBlock = () -> Void
+
+class URBNSwiftAccordionTableViewController: UITableViewController {
+    
+    var adapter: URBNAccordionDataSourceAdapter?
+    @IBOutlet var stepper: UIStepper?
```

Comment:
I think it makes sense in the world of IB. If the nib is not set up correctly the app should crash and you should feel bad